tests/test_tokenize.py

```diff
@@ -49,3 +51,45 @@ def test_tokenize_with_client_against_model(client: AlephAlphaClient, model_name
 
     assert len(response["tokens"]) == 1
     assert len(response["token_ids"]) == 1
+
+
+def test_offline_tokenizer_sync(sync_client: Client, model_name: str):
+    prompt = "Hello world"
+
+    tokenizer = sync_client.tokenizer(model_name)
+    offline_tokenization = tokenizer.encode(prompt)
+
+    tokenization_request = TokenizationRequest(
+        prompt=prompt, token_ids=True, tokens=True
+    )
+    online_tokenization_response = sync_client.tokenize(
+        tokenization_request, model_name
+    )
+
+    assert offline_tokenization.ids == online_tokenization_response.token_ids
+    assert offline_tokenization.tokens == online_tokenization_response.tokens
+
+    detokenization_request = DetokenizationRequest(
+        token_ids=online_tokenization_response.token_ids
+    )
+    online_detokenization_response = sync_client.detokenize(
+        detokenization_request, model_name
+    )
+
+    assert (
+        tokenizer.decode(offline_tokenization.ids)
+        == online_detokenization_response.result
+    )
+
+
+async def test_offline_tokenizer_async(async_client: AsyncClient, model_name: str):
+    prompt = "Hello world"
+
+    tokenizer = await async_client.tokenizer(model_name)
+    offline_tokenization = tokenizer.encode(prompt)
+
+    request = TokenizationRequest(prompt=prompt, token_ids=True, tokens=True)
```

Comment:
How about the detokenizer?

Reply:
didn't want to test that twice because it has nothing to do with the AsyncClient's functionality. After splitting the above test up, it makes even more sense.